backend/src/main/kotlin/org/loculus/backend/service/files/FilesTable.kt

```diff
@@ -0,0 +1,17 @@
+package org.loculus.backend.service.files
+
+import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.kotlin.datetime.datetime
+import java.util.*
+
+typealias FileId = UUID
+const val FILES_TABLE_NAME = "files"
+
+object FilesTable : Table(FILES_TABLE_NAME) {
+
+    val idColumn = uuid("id")
+    val uploadRequestedAtColumn = datetime("upload_requested_at")
+    val uploaderColumn = text("uploader")
+    val groupIdColumn = integer("group_id")
+    val publishedAtColumn = datetime("published_at").nullable()
```

Comment:
Technically this is denormalized if files are public when the corresponding entry is released - though that would require a bunch of unpleasant joining (finding out what the corresponding entry is and what its release date is)

Reply:
A file can be associated with multiple sequence entries, so it's practical to record the timestamp explicitly. Also, a processed file can be created and released after the approval of the sequence entry when the pipeline reruns and decides to create new files (#4055).